packages/react-meteor-data/suspense/useSubscribe.ts

```diff
@@ -0,0 +1,63 @@
+import { useEffect } from 'react'
+import { EJSON } from 'meteor/ejson'
+import { Meteor } from 'meteor/meteor'
+import isEqual from 'lodash/isEqual'
+import { removeFromArray } from './useFind'
+
+const cachedSubscriptions: SubscriptionEntry[] = []
+
+interface SubscriptionEntry {
+  params: EJSON[]
+  name: string
+  handle?: Meteor.SubscriptionHandle
+  promise: Promise<Meteor.SubscriptionHandle>
+  result?: Meteor.SubscriptionHandle
+  error?: unknown
+}
+
+export function useSubscribeSuspense(name: string, ...params: EJSON[]) {
+  const cachedSubscription =
+    cachedSubscriptions.find(x => x.name === name && isEqual(x.params, params))
+
+  useEffect(() => {
+    return () => {
```

Comment:
Minor stylistic change:
```suggestion
  useEffect(() => 
    () => {
```